src/ui/analyse/view/boardView.ts

```diff
@@ -8,6 +8,7 @@ import { Shape } from '../../shared/BoardBrush'
 import Clock from './Clock'
 import { povDiff } from '../ceval/winningChances'
 import AnalyseCtrl from '../AnalyseCtrl'
+import { opposite } from '~/chessground/util'
```

Comment:
chessops appears to have an identical `opposite` function - should we use that here? and if so, should we replace all instances of chessground/util opposite witth chessops opposite?